src/custom/simplifions/components/SimplifionsSolutionCard.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <div class="topic-card">
+    <RouterLink :to="topicLink">
+      <div class="header-topic">
+        <div class="title-topic">{{ topic.name }}</div>
+        <div class="author-topic">
+          <template v-if="topic.organization">
+            <OrganizationNameWithCertificate
+              :organization="topic.organization"
+            />
+          </template>
+          <template v-else>{{ ownerName }}</template>
+        </div>
+        <div class="date-topic">
+          mis à jour {{ formatRelativeIfRecentDate(topic.last_modified) }}
+        </div>
+      </div>
+    </RouterLink>
+    <div class="description-topic">
```

Comment:
Pourquoi ne pas rendre cette partie clickable aussi ?

Reply:
J'ai repris l'UI que Dorine (l'UI/UX designer) avait faite, je la laisserait itérer sur ces composants. Pour l'instant le but est d'avoir une version de base qui fonctionne, pour ensuite qu'elle puisse itérer en autonomie sur l'UI.